ts/src/profiler.ts

```diff
@@ -72,6 +73,21 @@ export interface RequestProfile {
   labels?: {instance?: string};
 }
 
+interface ServerBackoffResponse {
```

Comment:
Document that this corresponds to the server response carrying RetryInfo from https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto. Can we use the proto definition from that repo instead of defining the structure manually?

Reply:
Filed an issue -- I'd need to look more into it, and if it works, I think it would be best as a separate PR.